pkg/textparse/testdata.txt

```diff
@@ -525,4 +525,7 @@ prometheus_target_sync_length_seconds_count{scrape_job="prometheus"} 1
 prometheus_treecache_watcher_goroutines 0
 # HELP prometheus_treecache_zookeeper_failures_total The total number of ZooKeeper failures.
 # TYPE prometheus_treecache_zookeeper_failures_total counter
-prometheus_treecache_zookeeper_failures_total 0
\ No newline at end of file
+prometheus_treecache_zookeeper_failures_total 0
+# HELP redis_up help text
+# TYPE redis_up gauge
+redis_up{addr="redis://localhost:6379",alias=""} 1
```

Comment:
Not sure why you have these changes included.

Reply:
sorry, it is testing the wrong help text for redis-exporter, it should not appear here...

Reply:
I will remove it in next commit, sorry for that